### PR TITLE
{tools}[foss/2018b] bokeh v1.0.4 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bokeh/bokeh-1.0.4-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('PyYAML', '3.13', versionsuffix),
+]
+
+use_pip = True
+
```
Comment (vanzod) [inline easybuild/easyconfigs/b/bokeh/bokeh-1.0.4-foss-2018b-Python-3.6.6.eb]: Please also add `download_dep_fail = True`

Comment (smoors) [reply easybuild/easyconfigs/b/bokeh/bokeh-1.0.4-foss-2018b-Python-3.6.6.eb]: @vanzod `download_dep_fail = True` is already the default in `PythonBundle` easyblock.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2418.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/900563e64d5a872e8b3cce2c9739fda1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2014.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/da124b25d8d618f418c9edd45107be45 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3156.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c17d9b28bc028fd4aa8de1c28a4e5ac8 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**